app/src/main/java/com/rentminder/MainActivity.kt

```diff
@@ -62,18 +62,14 @@ fun MainMenu() {
 
     Column() {
         TopToolBar()
-        Row(modifier = Modifier
```

Comment:
Changed the formatting of these lines to have less line breaks. This makes the functions much more readable.